### DISCUSSION TITLE
What is SeaORM?

### DISCUSSION THREAD

**Discussion by ibraheemdev** [SeaQL/sea-query#9] [General]
Hey, this looks like a really cool project! The readme mentions that this is the foundation for upcoming ORM - is it going to be a high-level SQL ORM like diesel, or are you developing something else?

**Comment by billy1624** [comment]
SeaORM will feature atomic document read & write on relational databases like MySQL and PostgreSQL. Providing similar document concept in NoSQL databases such as MongoDB but the data will be stored across multiple tables in relational database.

**Comment by tyt2y3** [comment]
My 2 cents, I guess it will be more like TypeORM (we are heavily inspired) than diesel, meaning with features catered for web development in mind, like utilities for filtering, sorting and paging and many joins!

**Comment by ibraheemdev** [comment]
That sounds really cool! Good luck :+1:

**Comment by tyt2y3** [comment]
Now, really, SeaORM is released.
https://github.com/SeaQL/sea-orm

**Reply by ibraheemdev** [reply]
Looks very cool! You might want to write a release post and submit it to thisweekinrust.

**Reply by billy1624** [reply]
Ohhhh, nice suggestion!

**Reply by tyt2y3** [reply]
Thank you. It's been a long road. Yes, we are now testing it publicly. Definitely will announce it soon!
